docs/content/concepts/dagster-pipes/pyspark.mdx

```diff
@@ -0,0 +1,328 @@
+---
+title: "Integrating PySpark with Dagster Pipes | Dagster Docs"
+description: "Learn to integrate Dagster Pipes with PySpark to orchestrate PySpark jobs in a Dagster pipeline."
+---
+
+# PySpark & Dagster Pipes
+
+This tutorial is focused on using Dagster Pipes to launch & monitor general PySpark jobs. The [Spark integration page](/integrations/spark) provides more information on using Pipes with specific Spark provides (such as AWS EMR or Databricks).
+
+Spark is often used with object stores such as Amazon S3. We are going to simulate this setup locally with [MinIO](https://min.io/) which has an API compatible with AWS S3. All communication between Dagster and the Spark job (metadata and logs collection) will happen through a MinIO bucket.
+
+---
+
+## Prerequisites
+
+We are going to use [Docker](https://docs.docker.com/get-started/get-docker/) to emulate a typical Spark setup locally. Therefore, only Docker is required to reproduce this example.
+
+## Production considerations
+
+For demonstration purposes, this tutorial makes a few simplifications that you should consider when deploying in production:
+
+- We are going to be running Spark in local mode, sharing the same container with Dagster. In production, consider having two separate environments.
+
+   - **In the Dagster environment**, you'll need to have the following Python packages:
+
+     - `dagster`
+     - `dagster-webserver` --- to run the Dagster UI
+     - `dagster-aws` --- when using S3
+
+     And make the orchestration code available to Dagster (typically via a code location).
```

Comment:
```suggestion
     You will also need to make the orchestration code available to Dagster (typically via a code location).
```

Reply:
I would also recommend making "code location" a link to the code location docs (/concepts/code-locations).